racunaljka.js

```diff
@@ -0,0 +1,44 @@
+function provide(nameOfNewClass, classConstructor) {
```

Comment:
We are going into right direction, but we need to support multiple classes at the same time. The idea is that you register once via provide and than use it when you need it later in the code (multiple times):

```
provide('djani', ClassA);
provide('zile', ClassB);
provide('kungfu', ClassC);

// later in the code
var Kungfu = require('kungfu');
var Djani = require('djani');
```